packages/blocks/shuffle/src/shuffle.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useReducer, useRef } from "react";
+import {
+  BlockComponent,
+  useGraphBlockService,
+} from "@blockprotocol/graph/react";
+// eslint-disable-next-line no-restricted-imports
+import Button from "@mui/material/Button";
+import Box from "@mui/material/Box";
+import { ItemList } from "./components/item-list";
+import { ActionType, initialItems, Items, shuffleReducer } from "./reducer";
+
+type BlockEntityProperties = {
+  items: Items;
```

Comment:
The block can handle being loaded without items, so this can be optional - it also helps with spotting where the block might crash if `items` is not passed in
```suggestion
  items?: Items;
```

Reply:
Addressed in fa809e0d75e04a017291b2a8789a6f4392d05cf2